### PR TITLE
Release Notes for April 2024 Release SDK & Prerelease SDK

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article section for review:
* **Release Notes for the WebView2 SDK** > **1.0.2478.35**
   * https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/?branch=pr-en-us-3125#10247835
   * [After/Live](https://learn.microsoft.com/microsoft-edge/webview2/release-notes/#10247835)
* **Release Notes for the WebView2 SDK** > **1.0.2525-prerelease**
   * https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/?branch=pr-en-us-3125#102525-prerelease
   * [After/Live](https://learn.microsoft.com/microsoft-edge/webview2/release-notes/#102525-prerelease)

This PR also does the following:
* Removes trailing space char.
* Removes "General features" heading from template; that heading hasn't been used for 2 years and just was extra work to remove it (empty section) each month.
* Removes file "template-prerelease.md" & defines a redirect.  Test:
Expressed per redirection json:
   "source_path": "microsoft-edge/webview2/release-notes/template-prerelease.md",
Expressed per this PR branch:
https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/template-prerelease?branch=pr-en-us-3125
Result: Success; end up at top of RelNotes webpage:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes/?branch=pr-en-us-3125&tabs=dotnetcsharp

AB#50080267

### CODE REVIEW

File: microsoft-edge/webview2/release-notes/index.md [hunk 1]
```diff
@@ -67,6 +66,297 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
...
+
+* Fixed a bug in WinRT JavaScript projection where passing in a typed array resulted in an "Interface Not Supported" error.  ([Issue #3486](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3486))
+
+* Added support for handling `out` array parameters in WinRT JavaScript projection.
+
+
+<!-- ---------- -->
+###### Runtime-only
+
+* The Image Auto-captioning feature on the context menu is no longer enabled.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes/index.md:326]: pls rewrite in standard format, like:
Fixed a bug in which [incorrect behavior].  
Orig unclear text was:
Image Auto-captioning feature on context menu is disabled
@victorhuangwq , @peiche-jessica

File: microsoft-edge/webview2/release-notes/index.md [hunk 2]
```diff
@@ -67,6 +66,298 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
...
+  * [CoreWebView2.SaveAsUIShowing Event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-1.0.2525-prerelease&preserve-view=true#saveasuishowing)
+  * [CoreWebView2.ShowSaveAsUIAsync Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-1.0.2525-prerelease&preserve-view=true#showsaveasuiasync)
+
+* [CoreWebView2SaveAsKind Enum](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2saveaskind?view=webview2-winrt-1.0.2525-prerelease&preserve-view=true)
+   * `Complete`
+   * `Default`
+   * `HtmlOnly`
+   * `SingleFile`
+  
+* [CoreWebView2SaveAsUIResult Enum](/microsoft-
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes/index.md:214]: @victorhuangwq  - i'm fixing linebreak

File: microsoft-edge/webview2/release-notes/index.md [hunk 3]
```diff
@@ -67,6 +66,298 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
...
+   * [ICoreWebView2ExperimentalSaveAsUIShowingEventArgs::put_AllowReplace](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalsaveasuishowingeventargs?view=webview2-1.0.2525-prerelease&preserve-view=true#put_allowreplace)
+   * [ICoreWebView2ExperimentalSaveAsUIShowingEventArgs::get_AllowReplace](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalsaveasuishowingeventargs?view=webview2-1.0.2525-prerelease&preserve-view=true#get_allowreplace)
+   * [ICoreWebView2ExperimentalSaveAsUIShowingEventArgs::put_Kind](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalsaveasuishowingeventargs?view=webview2-1.0.2525-prerelease&preserve-view=true#put_kind)
+   * [ICoreWebView2ExperimentalSaveAsUIShowingEventArgs::get_Kind](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalsaveasuishowingeventargs?view=webview2-1.0.2525-prerelease&preserve-view=true#get_kind)
+   * [ICoreWebView2ExperimentalSaveAsUIShowingEventArgs::Invoke](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalsaveasuishowingeventargs?view=webview2-1.0.2525-prerelease&preserve-view=true#invoke)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes/index.md:268]: delete Invoke?  @peiche-jessica @victorhuangwq

File: microsoft-edge/webview2/release-notes/index.md [hunk 4]
```diff
@@ -67,6 +66,297 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
...
+   * [ICoreWebView2ExperimentalSaveAsUIShowingEventArgs::get_AllowReplace](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalsaveasuishowingeventargs?view=webview2-1.0.2525-prerelease&preserve-view=true#get_allowreplace)
+   * [ICoreWebView2ExperimentalSaveAsUIShowingEventArgs::put_Kind](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalsaveasuishowingeventargs?view=webview2-1.0.2525-prerelease&preserve-view=true#put_kind)
+   * [ICoreWebView2ExperimentalSaveAsUIShowingEventArgs::get_Kind](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalsaveasuishowingeventargs?view=webview2-1.0.2525-prerelease&preserve-view=true#get_kind)
+   * [ICoreWebView2ExperimentalSaveAsUIShowingEventArgs::Invoke](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalsaveasuishowingeventargs?view=webview2-1.0.2525-prerelease&preserve-view=true#invoke)<!-- todo: delete?  not at https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimentalsaveasuishowingeventargs?view=webview2-win32-preview&branch=pr-en-us-89#invoke -->
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes/index.md:267]: delete Invoke?  @peiche-jessica @victorhuangwq 
not found in:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimentalsaveasuishowingeventargs?view=webview2-win32-preview&branch=pr-en-us-89#invoke

File: microsoft-edge/webview2/release-notes/index.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes/index.md]:
```
* Fixed a bug where the Image Auto-captioning feature was enabled by default. 
This feature is no longer enabled.
```

Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes/index.md]: removing "This feature is no longer enabled." b/c sounds like it's completely gone now.  remainder is clearest, least confusing:
* Fixed a bug where the Image Auto-captioning feature was enabled by default.

File: microsoft-edge/webview2/release-notes/index.md (hunk 4 above)
Comment (peiche-jessica) [reply microsoft-edge/webview2/release-notes/index.md:267]: Please move the Invoke to ICoreWebView2ExperimentalShowSaveAsUICompletedHandler. `   * [ICoreWebView2ExperimentalShowSaveAsUICompletedHandler::Invoke](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalshowsaveasuicompletedhandler?view=webview2-1.0.2525-prerelease&preserve-view=true#invoke)`

Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes/index.md:267]: success: the expanded trimmed url ends up at:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimentalshowsaveasuicompletedhandler?view=webview2-win32-preview&branch=pr-en-us-89#invoke

**Review by victorhuangwq — APPROVED** [review]
LGTM

File: microsoft-edge/webview2/release-notes/index.md [hunk 5]
```diff
@@ -55,18 +54,309 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
...
+Release Date: April 22, 2024
+
+[NuGet package for WebView2 SDK 1.0.2525-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2525-prerelease)
+
+For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 125.0.2525.0 or higher.
+
+<!-- ------------------------------ -->
+#### Breaking changes
+
+The minimum .NET Framework version requirement for .NET WebView2, including WPF and WinForms controls, has been updated from 4.5 to 4.6.2.
```
Comment (victorhuangwq) [inline microsoft-edge/webview2/release-notes/index.md:164]: @mikehoffms breaking change requirement.

**Review by victorhuangwq — COMMENTED** [review]
add breaking change where we update the minimum requirement

File: microsoft-edge/webview2/release-notes/index.md (hunk 5 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes/index.md:164]: changed for better Find support:
```
The minimum .NET Framework version requirement for .NET WebView2, including WPF and WinForms controls, has been updated from .NET Framework 4.5 to .NET Framework 4.6.2.
```

**Pull request merged**